---
string[] label_names
float32[] prob
